src/cmd.rs

```diff
@@ -590,7 +659,7 @@ pub fn osd_crush_remove(cluster_handle: &Rados, osd_id: u64, simulate: bool) ->
 }
 
 /// Query a ceph pool.
-pub fn osd_pool_get(cluster_handle: &Rados, pool: &str, choice: &PoolOption) -> RadosResult<String> {
+pub fn osd_pool_get(cluster_handle: &Rados, pool: &str, choice: PoolOption) -> RadosResult<String> {
```

Comment:
Unlike the (many) comments about moves vs pointers, the ones in this file are the reverse?